lib/hubspot/association.rb

```diff
@@ -75,6 +75,15 @@ def all(object_type, object_id, to_object_type)
       response['results'].map { |result| klass.find(result["toObjectId"]) }
     end
 
+    # Utility function to build the association list required by some API endpoints
+    def build_association_param(origin, associated, associated_id)
+      {
+        "to": { "id": associated_id },
+        "types": [{ "associationCategory": 'HUBSPOT_DEFINED',
+                    "associationTypeId": Hubspot::Association::ASSOCIATION_DEFINITIONS[origin][associated] }]
```

Comment:
seeing `build_create_association_body`, I'm pretty sure you can use symbols for keys
```suggestion
        to: { id: associated_id },
        types: [{ associationCategory: 'HUBSPOT_DEFINED',
                    associationTypeId: Hubspot::Association::ASSOCIATION_DEFINITIONS[origin][associated] }]
```